### PR TITLE
Added HTML only example for the video element.

### PR DESCRIPTION
**Pull request by iandevlin**
Added a HTML only example for the video element (issue https://github.com/w3c/html/issues/1316)

### CODE REVIEW

**Review by edent — APPROVED** [review]

File: sections/semantics-embedded-content.include [hunk 1]
```diff
@@ -5867,6 +5867,21 @@ attribute's value is a type that a <a>plugin</a> supports, then the value of the
   The <dfn attribute for="HTMLVideoElement"><code>poster</code></dfn> IDL attribute must
   <a>reflect</a> the <{video/poster}> content attribute.
 
+  <div class="example">
+    This example shows how different video files can be offered to the browser. If the browser does not support a specific codec,
+    it can play one of the alternative files offered.
+
+    This example also shows the <code>controls</code>, and <code>preload</code> attributes.
```
Comment (chaals) [inline sections/semantics-embedded-content.include]: The attributes should be linked here, using `<{video/controls}>` and `<{video/preload}>` syntax.

**Review by chaals — CHANGES_REQUESTED** [review]
It would be helpful to link the attributes you mention (something it seems we should do in more places in this chapter), but otherwise this looks good - thank you.

File: sections/semantics-embedded-content.include (hunk 1 above)
Comment (iandevlin) [reply sections/semantics-embedded-content.include]: Ah, I copied the example from `audio` below, and it didn't have that. Will correct, and the `audio` example too while I am at it, if that is ok?

Comment (chaals) [reply sections/semantics-embedded-content.include]: yes, doing that would be great - thank you!

**Review by chaals — APPROVED** [review]

**Pull request merged**